libminifi/test/TestBase.h

```diff
@@ -304,7 +323,7 @@ class TestController {
             std::string file(dir);
             file += "/";
             file += dir_entry->d_name;
-            unlink(file.c_str());
+//            unlink(file.c_str());
```

Comment:
This doesn't look intentional/final.

Reply:
Please note the "draft" state of the PR :)